### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by Mykyta-Pysarev**
Search form for right sidebar

### CODE REVIEW

File: src/containers/Search.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Search as SearchIcon } from "react-bootstrap-icons";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
```
Comment (KirillSerg) [inline src/containers/Search.tsx:3]: Why  you dont use destructuring like "import {Form,  InputGroup} from "react-bootstrap";"?
one line, less code.

File: src/containers/Search.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import styled from "styled-components";
+
+const StyledForm = styled(Form)`
+  border: 2px solid white;
```
Comment (KirillSerg) [inline src/containers/Search.tsx:7]: it seems there is class"border" in the bootstrap

**Review by KirillSerg — APPROVED** [review]
cool

File: src/containers/Search.tsx (hunk 1 above)
Comment (Mykyta-Pysarev) [reply src/containers/Search.tsx:3]: Import in VS code works as default like this.

**Comment by Mykyta-Pysarev** [comment]
image

**Pull request closed without merging**